web/controllers/respondent_controller.ex

```diff
@@ -65,11 +86,13 @@ defmodule Ask.RespondentController do
     {:ok, local_time } = Ecto.DateTime.cast :calendar.local_time()
 
     if Path.extname(file.filename) == ".csv" do
-      entries = File.stream!(file.path) |>
-      CSV.decode(separator: ?\t) |>
-      Enum.map(fn row ->
-        %{phone_number: Enum.at(row, 0), survey_id: integer_survey_id, inserted_at: local_time, updated_at: local_time}
-      end)
+      csv_string = File.read!(file.path)
+      rows = csv_rows(csv_string)
```

Comment:
if the variable `rows` is not used anywhere else, I would replace it by the function call:
``` entries = 
   csv_rows(csv_string)
   |>...```
or maybe
```entries =
        File.read!(file.path)
        |> csv_rows
        |>...```